handlers/getakt.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Module for requesting documents via AktDirekt, Lantmäteriet
+ * Usage: 
+ * Check for service health, response "UP" or "DOWN" - /origoserver/healthcheck
+ * Get a djvu index page that gets all pages included in akt(document) - /origoserver/document/index.djvu?archive={archive/county-number}&id={akt-number}
+ * Get a single djvu page - /origoserver/document/page_{vers}_{subdoc}_{page}_{archive}_{id}.djvu
+ * 
+ */
+
+// Dependencies
+const url = require('url');
+const http = require('http');
+const httpProxy = require('http-proxy');
+const config = require('../conf/config');
+const getToken = require('../lib/tokenrequest');
+
+// Use the config for AktDirekt
+const akturl = config.getAkt.url;
+const aktkey = config.getAkt.consumer_key;
+const aktsecret = config.getAkt.consumer_secret;
+const aktscope = config.getAkt.scope;
+
+// Token holder
+let token;
+
+// Proxy server holder
+let proxy = httpProxy.createProxyServer();
+
+// Get hold of that nested token and use in proxy request header
+function tokenHeader() { };
+
+// Proxy request config
+proxy.on('proxyReq', (proxyReq, req) => {
+
+  // Get the request path
+  const parsedUrl = url.parse(req.url);
+  const query = parsedUrl.search;
+  const path = parsedUrl.pathname;
+  const pathPart = path.split("_");
+
+  const pathObj = {
+    page_: pathPart[0],
+    vers: pathPart[1],
+    subdoc: pathPart[2],
+    page: pathPart[3],
+    archive: pathPart[4],
+    enc_id: pathPart[5]
+  }
+
+  // Proxy path holder
+  let proxyPath;
+
+  if (path.includes('healthcheck')) {
+    const healthcheck = `/distribution/produkter/aktdirekt/v3.0${path}`;
```

Comment:
"/distribution/produkter/aktdirekt/v3.0" could be a base url in config options or a variable, which might be useful in the future.

Reply:
Yes, I will variablize it.